cadquery/occ_impl/shapes.py

```diff
@@ -417,6 +417,8 @@ def exportStl(
         tolerance: float = 1e-3,
         angularTolerance: float = 0.1,
         ascii: bool = False,
+        isToleranceRelative: bool = True,
+        isParallel: bool = False,
```

Comment:
For consistency change parameter name to `parallel`?  See _bool_op.

Reply:
Will do (tomorrow)